test/unit/supervisor_test.rb

```diff
@@ -48,6 +48,14 @@ class SupervisorTest < ActiveSupport::TestCase
     assert_not process_exists?(pid)
   end
 
+  test "start with invalid configuration" do
```

Comment:
I wasn't entirely sure how to test the abort message. Perhaps that's OK though? There are no other tests for the specific abort message.

Reply:
Ah! A way forward on making this test a little better: https://docs.seattlerb.org/minitest/Minitest/Assertions.html#method-i-capture_subprocess_io